languages/kotlin/config.toml

```diff
@@ -3,5 +3,7 @@ grammar = "kotlin"
 path_suffixes = ["kt"]
 line_comments = ["// "]
 brackets = [
-		{ start = "(", end = ")", close = true, newline = false}
+		{ start = "(", end = ")", close = true, newline = false },
+		{ start = "{", end = "}", close = true, newline = false },
+	    { start = "<", end = ">", close = true, newline = false },
```

Comment:
I'm not an expert, but won't autoclosing `<` interfere with comparison operators?

Reply:
Yeah, this config is not contextual; we have pretty rudimentary ways to exclude rules (see [rust config](https://github.com/zed-industries/zed/blob/main/crates/languages/src/rust/config.toml#L10) where we exclude autobrackets from strings and comment contexts), but nothing like "when in generic context".

Reply:
Some of our languages have <> configured and [we've gotten some complaints about it](https://github.com/zed-industries/zed/issues/10512).